### PR TITLE
[Neo4j] updating readme to reference stable instead of incubator

### PR DESCRIPTION
**Pull request by mneedham**

### CODE REVIEW

**Comment by mneedham** [comment]
@unguiculus I realised I'd forgotten to update the readme to use `stable` instead of `incubator`

File: stable/neo4j/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: neo4j
 home: https://www.neo4j.com
-version: 0.3.0
+version: 0.4.0
```
Comment (unguiculus) [inline stable/neo4j/Chart.yaml]: A patch bump should be sufficient: 0.3.1

**Pull request merged**